pkgs/applications/misc/stylish/default.nix

```diff
@@ -30,10 +50,10 @@ stdenvNoCC.mkDerivation rec {
       of finding new wallpapers, downloading and switching them via the
       configs.
       Styli.sh can search for specific wallpapers from unsplash or download
-      a random image from the specified subreddits. If you have pywal it also
-      can set automatically your terminal colors.
+      a random image from the specified subreddits. If you have pywal
+      it also can set automatically your terminal colors.
     '';
     license = licenses.mit;
-    maintainers = with maintainers; [ tchab ];
+    maintainers = with maintainers; [tchab];
```

Comment:
Hmm I think it's because I tested [a new formatter](https://github.com/kamadorueda/alejandra) and forgot to switch back to nixfmt, sorry for that. 

I will reformat the file.